### PR TITLE
Visitor can see an list of articles

### PR DESCRIPTION
**Pull request by FlisAnn**
[Visitor can see an list of articles ](https://www.pivotaltracker.com/story/show/176041986)
### User Story
```
As a Visitor
In order to choose an article to read
I would like to be able to see the articles listed on a page
```
## Changes proposed in this pull request:
*  Creating test for viewing articles
* Creating fixture file
* Creating DisplayArticle component
* Creating getArticle module 
* Using semantic-ui for styling 

## What I have learned working on this feature:
* Using useState hook
* Using useEffect
* All capital letters URL
* Don't remove things you're not sure about
## Packages/Gems added
* Axios
* Semantic-ui
## Screenshots (Client)
screen2

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (emtalen) [inline cypress/fixtures/example.json]: remove this file plz

File: cypress/integration/userCanSeeAListOfArticles.feature.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/articles",
+        response: "fixture:articles_data.json",
+      });
+      cy.visit("/");
+    });
+
+    it("successfully", () => {
```
Comment (emtalen) [inline cypress/integration/userCanSeeAListOfArticles.feature.js:13]: Is there a case for unsuccessfully?

File: src/index.js [hunk 3]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import axios from 'axios';
+import "semantic-ui-css/semantic.min.css"
+
+axios.defaults.baseURL = "http://localhost:3000/api/"
```
Comment (emtalen) [inline src/index.js]: have you deployed your api? in that case please make sure that you add the right URL

File: cypress/fixtures/example.json (hunk 1 above)
Comment (psnoya) [reply cypress/fixtures/example.json]: Done!

File: cypress/integration/userCanSeeAListOfArticles.feature.js (hunk 2 above)
Comment (psnoya) [reply cypress/integration/userCanSeeAListOfArticles.feature.js:13]: Now there is!

File: src/index.js (hunk 3 above)
Comment (psnoya) [reply src/index.js]: done

**Review by emtalen — APPROVED** [review]

**Pull request merged**